python/pyspark/pandas/frame.py

```diff
@@ -11195,29 +11195,55 @@ def any(
         >>> df[[]].any()
         Series([], dtype: bool)
         """
-        axis = validate_axis(axis)
```

Comment:
Do we still want to call validate_axis to convert it to a numeric ref (otherwise the axis = 0 condition might miss something)